### PR TITLE
A sketch of a framework for our library

### PR DESCRIPTION
**Pull request by amenshov**
Changes:
- remove bower
- add basic blocks
- add example

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 bower_components/
```
Comment (102) [inline .gitignore]: May be we should remove `bower_components/` from gitignore if we wouldn't use it anymore?

Comment (amenshov) [inline .gitignore]: right!

File: frontend/blocks/actions/actions.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    constructor(parent, config) {
+        super(parent, config);
+
+        this._initItems();
+    }
+
+    _initItems() {
+        let actions = this;
+
+        this._items = _.map(this._config.items, block_config => {
```
Comment (102) [inline frontend/blocks/actions/actions.js]: I think we can use `this._config.items.map(...)` here

File: frontend/blocks/actions/actions.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    get templateFn() {
+        return template;
+    }
+
+    render() {
+        this.el = $(this.templateFn());
+
+        let actions = this;
+
+        _.forEach(this._items, block => {
```
Comment (102) [inline frontend/blocks/actions/actions.js]: Same here: `this._items.forEach`

**Pull request merged**